apps/application-system/api/migrations/20220112162352-pruneAt-and-prune-index.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     return queryInterface.sequelize.query(`
       BEGIN;
 
-        CREATE INDEX pruned_and_pruneAt_idx ON application (prune_at) WHERE (pruned = FALSE AND prune_at IS NOT NULL);
+        CREATE INDEX pruned_and_pruneAt_idx ON application (prune_at, pruned) WHERE (pruned = FALSE AND prune_at IS NOT NULL);
```

Comment:
Sorry, forgot to mention that `pruned` needs to come before `prune_at`. Basically any property you filter for a specific value (`pruned = FALSE`) should come before properties that you filter by a range (`prune_at < now()`)